tests/test_mc_anchors.py

```
@@ -162,3 +175,67 @@ def test_serial_parallel_equivalence():
+    raw_ratio = pnl_total / STARTING_EQUITY
+    assert raw_ratio > DAILY_LOSS_PCT, (
+        f"Construction invariant: raw FP must land above -0.05 (less negative); "
+        f"got pnl_total={pnl_total!r}, ratio={raw_ratio!r}"
+    )
+    assert round(raw_ratio, 6) <= DAILY_LOSS_PCT, (
+        f"Construction invariant: round6 must collapse to -0.05 and fire; "
+        f"got round6={round(raw_ratio, 6)!r}"
+    )
+
+    # Run _simulate_path directly (NOT through compute_default_config).
+    outcome, day, max_dd, culprit = _simulate_path(
```

Ruff (0.15.12)

[warning] Unpacked variable `max_dd` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)